package.json

```diff
@@ -73,7 +73,7 @@
   "name": "zigbee-herdsman",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Koenkk/zigbee-herdsman.git"
+    "url": "git+https://github.com/Asgothian/zigbee-herdsman.git"
```

Comment:
Please revert this change

Reply:
Sorry, this change slipped through, will be changed back.